src/Framework/DataProviderTestSuite.php

```diff
@@ -17,34 +17,44 @@
 final class DataProviderTestSuite extends TestSuite
 {
     /**
-     * @var string[]
+     * @var array<string>
      */
     private $dependencies = [];
 
     /**
-     * @param string[] $dependencies
+     * @param array<string> $dependencies
      */
     public function setDependencies(array $dependencies): void
     {
-        $this->dependencies = $dependencies;
+        $this->dependencies  = $dependencies;
+        $this->requiredTests = TestUtil::trimDependencyOptions($dependencies);
 
         foreach ($this->tests as $test) {
             if (!$test instanceof TestCase) {
+                // @codeCoverageIgnoreStart
                 continue;
+                // @codeCoverageIgnoreStart
             }
-
             $test->setDependencies($dependencies);
         }
     }
 
-    public function getDependencies(): array
+    /**
+     * @return array<string>
```

Comment:
What do you think about omitting the DocBlock?

It will be inherited from the parent class.

Reply:
Hang on, `DataProviderTestSuite::getDependencies()` doesn't exist anymore... what diff is the pull request showing?

Reply:
Otherwise yes, `@inheritDoc` or automagic is fine with me

Reply:
figured out that Github shows an older version of the source in the inline view here, when clicking through to the full review it shows a newer version.